src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Form.js

```diff
@@ -150,7 +151,15 @@ class Form extends React.Component<Props> {
         } = store;
 
         if (store.forbidden) {
-            return <PermissionHint />;
+            return <Hint icon="su-lock" title={translate('sulu_admin.no_permissions')} />;
+        }
+
+        if (store.notFound) {
+            return <Hint icon="su-battery-low" title={translate('sulu_admin.not_found')} />;
+        }
+
+        if (store.unexpectedError) {
+            return <Hint icon="su-exclamation-triangle" title={translate('sulu_admin.unexpected_error')} />;
```

Comment:
Our UI consulting provided the feedback that that maybe this icons would be better match:

<img width="733" alt="Bildschirmfoto 2023-06-29 um 13 03 35" src="https://github.com/sulu/sulu/assets/1698337/0accc7e8-3f47-4a18-8d17-8cffbac9cdad">

Reply:
If we take one of these, I would use the sad smiley, but for now I don't think it will be a big issue to use the exclamation-triangle 😂